release-team/role-handbooks/ci-signal/README.md

```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
   - To get a better and quicker feedback on upgrade/downgrade tests, we can look at upgrade jobs that have "parallel" in their title eg: [gce-1.10-master-upgrade-cluster-parallel](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.10-master-upgrade-cluster-parallel). These complete faster and are more stable since they don't run slow long running tests. If the parallel upgrade/downgrade jobs are green, this indicates we are mostly good with respect to upgrade functionality
   - Yet another tip to assess seriousness of the failure is to see if both gce and gke counterparts of a upgrade/downgrade job are failing eg: [gce-1.10-master-upgrade-master](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.10-master-upgrade-master) and [gke-gci-1.10-gci-master-upgrade-master](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gke-gci-1.10-gci-master-upgrade-master). If both are failing then its indicative of a systemic failure in upgrade functionality. If only gke jobs are failing then it might be a GKE only issue and does not block k8s release. In this case escalate to GKE Cluster Lifecycle team  (@roberthbailey, @krousey)
 
+## Working with SIGs outside sig-release
+2 scenarios that you will be involved in:
+
+1. Identifying tests from sig-<name> that should be/are part of sig-release's blocking and informing dashboards. Those tests could be submitted as part of a new feature/enhancement that sig-<name> is developing, or could be existing tests in blocking/informing dashboards.
+Questions to ask sig-<name>:
+  - Which e2e test jobs are release blocking for your SIG?
```

Comment:
Most of them won't know, but I get the line of questioning.  In the case of catchall jobs (eg [release-master-blocking/gce-cos-master-default](https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-default)), I suspect we can at best ask that they are aware whether their sig has tests that run as part of those jobs.